.github/workflows/build.yml

```diff
@@ -0,0 +1,49 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-java-with-maven
+
+name: arquillian-extension-warp build
+
+on:
+  pull_request:
+    branches:
+      - master
+      - main
+    types: [opened, synchronize, reopened, ready_for_review]
+
+jobs:
+  build-test-matrix:
+    name: ${{ matrix.jdk-distribution}}-${{ matrix.jdk-version }}-${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 15
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+        jdk-distribution: [temurin]
+        jdk-version: [8]
+    steps:
+    - name: Configure runner - Linux
+      if: contains(matrix.os, 'ubuntu')
+      run: |
+        sudo bash -c "echo '127.0.0.1   localhost localhost.localdomain localhost4 localhost4.localdomain4' > /etc/hosts"
+        sudo bash -c "echo '::1         localhost localhost.localdomain localhost6 localhost6.localdomain6' >> /etc/hosts"
+    - name: Configure Runner - Windows
+      if: contains(matrix.os, 'windows')
+      run: |
+        echo '127.0.0.1   localhost localhost.localdomain localhost4 localhost4.localdomain4' > %SystemRoot%\System32\drivers\etc\hosts
+        echo '::1         localhost localhost.localdomain localhost6 localhost6.localdomain6' >> %SystemRoot%\System32\drivers\etc\hosts
+      shell: cmd
```

Comment:
Could you add a comment explaining why exactly is the host mapping needed?

Reply:
According to Ken from WildFly, the default hosts file in GitHub has a lot of hosts mapped to localhost (spam sites, Bitcoin mining sites etc) and it caused some weird behaviour in some tests.  gethostbyname("localhost") can return a random entry from /etc/hosts. That's why these steps are needed in building and running tests in WildFly. But I think it's less of a concern for this project, so I've removed these steps to simplify it a bit.